test/language/expressions/arrow-function/arrow/arrow-function-concise-1.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: arrow function concise body - assignment expression
+es6id: 14.2
+---*/
+
+var testVector = [
+    x => {x++},
+    x => {++x},
+    x => {!x},
+    x => {x|0},
+    x => {x && x},
+    x => {x || x},
+    x => {x^x},
+    x => {~x},
+    x => {x >> 1},
+    x => {x / x},
+    x => {x * 1},
+    x => {x % 1},
+    x => {+x}
+ ];
+
+ for(var i = 0; i<testVector.length; i++){
+
+     var arrowFn = testVector[i];
+
+     assert.sameValue((arrowFn instanceof Function), true, "Arrow function instanceof Function test failed for " + arrowFn.toString() + ".");
+
+     assert.sameValue(typeof arrowFn, "function", "Arrow function typeof failed for " + arrowFn.toString() + ".");
+
+     assert.sameValue(arrowFn.__proto__, Function.__proto__, "Arrow function __proto__ test failed for " + arrowFn.toString() + ".");
+
+     assert.sameValue(arrowFn.constructor, Function.constructor, "Constructor of arrow function is incorrect for " + arrowFn.toString() + ".");
+
+     assert.sameValue(('prototype' in arrowFn), false, "Arrow functions has prototype property for " + arrowFn.toString() + ".");
+
+     assert.sameValue(arrowFn.hasOwnProperty('prototype'), false, "Arrow functions do not have own property \'prototype\' for " + arrowFn.toString() + ".");
+
+     assert.sameValue(Object.getPrototypeOf(arrowFn), Function.prototype, "Object.getPrototypeOf arrowFn failed for arrow function for " + arrowFn.toString() + ".");
+}

```

Comment:
This file is duplicating tests in https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/prototype-rules.js